core/templates/services/question-validation.service.ts

```diff
@@ -94,6 +122,17 @@ export class QuestionValidationService {
     if (!atLeastOneAnswerCorrect) {
       return 'At least one answer should be marked correct';
     }
+    if (
+      question._stateData.interaction.answerGroups.length > 1 ||
+      question._stateData.interaction.answerGroups[0].rules.length > 1
+    ) {
+      if (this.hasDuplicateRules(question)) {
+        return (
+          'Duplicate rule detected. Ensure that no two rules in any answer group ' +
```

Comment:
Please make this message more specific so the lesson creator knows exactly which rules to fix.